### PR TITLE
RavenDB-18288 - Sharding - Conflicts View Endpoints

### PR DESCRIPTION
**Pull request by shiranshalom**
### Issue link

https://issues.hibernatingrhinos.com/issue/RavenDB-18288

### Type of change

- New feature

### How risky is the change?

- Not relevant

### Backward compatibility

- Not relevant

### Is it platform specific issue?

- No

### Documentation update

- No documentation update is needed 

### Testing 

- Tests have been added that prove the fix is effective or that the feature works

### Is there any existing behavior change of other features due to this change?

- No

### UI work

- No UI work is needed

### CODE REVIEW

File: src/Raven.Server/Documents/Sharding/Handlers/Processors/Replication/ShardedReplicationHandlerProcessorForGetConflicts.cs [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+            await RequestHandler.ShardExecutor.ExecuteSingleShardAsync(cmd, shardNumber);
+        }
+
+        private async Task WriteConflictsByEtag(JsonOperationContext context, BlittableJsonReaderObject[] conflicts, long totalResults)
+        {
+            await using (var writer = new AsyncBlittableJsonTextWriter(context, RequestHandler.ResponseBodyStream()))
+            {
+                context.Write(writer, new DynamicJsonValue
+                {
+                    ["TotalResults"] = totalResults,
```
Comment (karmeli87) [inline src/Raven.Server/Documents/Sharding/Handlers/Processors/Replication/ShardedReplicationHandlerProcessorForGetConflicts.cs]: add this to the class and use nameof

File: src/Raven.Server/Documents/Operations/GetReplicationConflictsOperation.cs [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+
+        internal class GetReplicationConflictsCommand : RavenCommand<BlittableArrayResult>
+        {
+            private readonly string _documentId;
+            private readonly long _etag;
+
+            public override bool IsReadRequest => true;
+
+            public GetReplicationConflictsCommand(string documentId = null, long etag = default)
+            {
```
Comment (karmeli87) [inline src/Raven.Server/Documents/Operations/GetReplicationConflictsOperation.cs]: you need also to pass `start` & `pageSize` here no?

File: src/Raven.Server/Documents/Operations/GetReplicationConflictsOperation.cs [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+using Raven.Client.Documents.Conventions;
+using Raven.Client.Documents.Operations;
+using Raven.Client.Http;
+using Raven.Client.Json;
+using Raven.Server.Json;
+using Sparrow.Json;
+
+namespace Raven.Server.Documents.Operations
+{
+    internal class GetReplicationConflictsOperation : IMaintenanceOperation<BlittableArrayResult>
```
Comment (karmeli87) [inline src/Raven.Server/Documents/Operations/GetReplicationConflictsOperation.cs]: let's avoid `BlittableArrayResult` and use something like
```
public class ConflictResults
{
 public ConflictResult[] Results;
 public long TotalResults;
}

public class ConflictResult
{
   public string Id;
   public string LastModified;
   ...
}
```

Comment (karmeli87) [reply src/Raven.Server/Documents/Operations/GetReplicationConflictsOperation.cs]: I think we already have this here `GetConflictsResult`

File: src/Raven.Server/Documents/Sharding/Commands/GetDocumentConflictsCommand.cs [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+using System.Net.Http;
+using System.Threading.Tasks;
+using Raven.Client.Documents.Commands;
+using Raven.Client.Http;
+using Raven.Server.Documents.Sharding.Handlers;
+using Sparrow.Json;
+
+namespace Raven.Server.Documents.Sharding.Commands
+{
+    internal class GetDocumentConflictsCommand : ShardedStreamCommand
```
Comment (karmeli87) [inline src/Raven.Server/Documents/Sharding/Commands/GetDocumentConflictsCommand.cs]: let's kill this and simply use the `GetReplicationConflictsCommand` instead

File: src/Raven.Server/Documents/Sharding/Handlers/Processors/Replication/ShardedReplicationHandlerProcessorForGetConflicts.cs [hunk 5]
```diff
@@ -0,0 +1,107 @@
...
+        {
+            private readonly HttpContext _httpContext;
+            private readonly TransactionOperationContext _context;
+            private readonly long _etag;
+
+            public ShardedGetReplicationConflictsOperation(HttpContext httpContext, TransactionOperationContext context, long etag)
+            {
+                _httpContext = httpContext;
+                _context = context;
+                _etag = etag;
```
Comment (karmeli87) [inline src/Raven.Server/Documents/Sharding/Handlers/Processors/Replication/ShardedReplicationHandlerProcessorForGetConflicts.cs]: You don't use here pageSize/start + etag is local per database.
You will probably need to use the `ShardedPagingContinuation` here.
ping me about that.

Comment (karmeli87) [reply src/Raven.Server/Documents/Sharding/Handlers/Processors/Replication/ShardedReplicationHandlerProcessorForGetConflicts.cs]: Let's block passing etag for the sharded EP entirely and use only the `ShardedPagingContinuation`.
note that we need allow paging here so don't throw on this function entirely.

File: src/Raven.Server/Documents/Sharding/Handlers/Processors/Replication/ShardedReplicationHandlerProcessorForGetConflicts.cs [hunk 6]
```diff
@@ -0,0 +1,107 @@
...
+            }
+
+            public HttpRequest HttpRequest => _httpContext.Request;
+
+            public BlittableJsonReaderObject[] Combine(Memory<BlittableArrayResult> results)
```
Comment (karmeli87) [inline src/Raven.Server/Documents/Sharding/Handlers/Processors/Replication/ShardedReplicationHandlerProcessorForGetConflicts.cs]: let's avoid `BlittableArrayResult` here as well

Comment (karmeli87) [reply src/Raven.Server/Documents/Sharding/Handlers/Processors/Replication/ShardedReplicationHandlerProcessorForGetConflicts.cs]: and everywhere else :)

File: src/Raven.Server/Web/Studio/Sharding/Commands/GetConflictResolutionCommand.cs [hunk 7]
```diff
@@ -0,0 +1,35 @@
...
+using System.Threading.Tasks;
+using Raven.Client.Http;
+using Raven.Server.Documents.Sharding.Commands;
+using Raven.Server.Documents.Sharding.Handlers;
+using Raven.Server.Web.Operations;
+using Sparrow.Json;
+
+namespace Raven.Server.Web.Studio.Sharding.Commands
+{
+    internal class GetConflictResolutionCommand : ShardedStreamCommand
```
Comment (karmeli87) [inline src/Raven.Server/Web/Studio/Sharding/Commands/GetConflictResolutionCommand.cs]: I don't think we need that if we would use the `AbstractHandlerProxyReadProcessor` instead

File: src/Raven.Server/Web/Studio/Processors/AbstractStudioDatabaseTasksHandlerProcessorForGetSuggestConflictResolution.cs [hunk 8]
```diff
@@ -0,0 +1,27 @@
+﻿using System.Threading.Tasks;
+using JetBrains.Annotations;
+using Raven.Server.Documents.Handlers.Processors;
+using Sparrow.Json;
+
+namespace Raven.Server.Web.Studio.Processors
+{
+    internal abstract class AbstractStudioDatabaseTasksHandlerProcessorForGetSuggestConflictResolution<TRequestHandler, TOperationContext> : AbstractHandlerProcessor<TRequestHandler, TOperationContext>
```
Comment (karmeli87) [inline src/Raven.Server/Web/Studio/Processors/AbstractStudioDatabaseTasksHandlerProcessorForGetSuggestConflictResolution.cs:8]: inherit from `AbstractHandlerProxyReadProcessor`

**Review by karmeli87 — CHANGES_REQUESTED** [review]

File: src/Raven.Server/Web/Studio/Sharding/Commands/GetConflictResolutionCommand.cs (hunk 7 above)
Comment (ppekrol) [reply src/Raven.Server/Web/Studio/Sharding/Commands/GetConflictResolutionCommand.cs]: Ping me tomorrow about this

File: src/Raven.Client/Documents/Commands/GetConflictsCommand.cs [hunk 9]
```diff
@@ -1,11 +1,28 @@
...
 using System.Net.Http;
+using Raven.Client.Documents.Conventions;
+using Raven.Client.Documents.Operations;
 using Raven.Client.Http;
 using Raven.Client.Json.Serialization;
 using Sparrow.Json;
 
 namespace Raven.Client.Documents.Commands
 {
+    public class GetConflictsOperation : IMaintenanceOperation<GetConflictsResult>
```
Comment (ppekrol) [inline src/Raven.Client/Documents/Commands/GetConflictsCommand.cs]: avoid adding public commands to the Client API if not needed

File: src/Raven.Server/Documents/Sharding/Handlers/Processors/Replication/ShardedReplicationHandlerProcessorForGetConflicts.cs (hunk 1 above)
Comment (shiranshalom) [reply src/Raven.Server/Documents/Sharding/Handlers/Processors/Replication/ShardedReplicationHandlerProcessorForGetConflicts.cs]: done

File: src/Raven.Server/Documents/Operations/GetReplicationConflictsOperation.cs (hunk 2 above)
Comment (shiranshalom) [reply src/Raven.Server/Documents/Operations/GetReplicationConflictsOperation.cs]: done

File: src/Raven.Server/Documents/Operations/GetReplicationConflictsOperation.cs (hunk 3 above)
Comment (shiranshalom) [reply src/Raven.Server/Documents/Operations/GetReplicationConflictsOperation.cs]: done

File: src/Raven.Server/Documents/Sharding/Commands/GetDocumentConflictsCommand.cs (hunk 4 above)
Comment (shiranshalom) [reply src/Raven.Server/Documents/Sharding/Commands/GetDocumentConflictsCommand.cs]: done

File: src/Raven.Server/Documents/Sharding/Handlers/Processors/Replication/ShardedReplicationHandlerProcessorForGetConflicts.cs (hunk 5 above)
Comment (shiranshalom) [reply src/Raven.Server/Documents/Sharding/Handlers/Processors/Replication/ShardedReplicationHandlerProcessorForGetConflicts.cs]: done

File: src/Raven.Server/Documents/Sharding/Handlers/Processors/Replication/ShardedReplicationHandlerProcessorForGetConflicts.cs (hunk 6 above)
Comment (shiranshalom) [reply src/Raven.Server/Documents/Sharding/Handlers/Processors/Replication/ShardedReplicationHandlerProcessorForGetConflicts.cs]: done

File: src/Raven.Client/Documents/Commands/GetConflictsResult.cs [hunk 10]
```diff
@@ -20,4 +22,20 @@ public class Conflict
             public BlittableJsonReaderObject Doc { get; set; }
         }
     }
+
+    public class GetConflictsResultByEtag
```
Comment (karmeli87) [inline src/Raven.Client/Documents/Commands/GetConflictsResult.cs]: This is not longer by etag right? it is more `GetConflictsPreviewResult`

File: src/Raven.Client/Documents/Commands/GetConflictsResult.cs [hunk 11]
```diff
@@ -20,4 +22,20 @@ public class Conflict
...
+    {
+        public ResultByEtag[] Results { get; internal set; }
+
+        public long TotalResults { get; set; }
+
+        public string ContinuationToken { get; set; }
+
+        public class ResultByEtag
```
Comment (karmeli87) [inline src/Raven.Client/Documents/Commands/GetConflictsResult.cs]: `ConflictPreview`

File: src/Raven.Server/Documents/Operations/GetReplicationConflictsOperation.cs [hunk 12]
```diff
@@ -0,0 +1,107 @@
...
+using Raven.Client.Documents.Conventions;
+using Raven.Client.Documents.Operations;
+using Raven.Client.Http;
+using Raven.Server.Documents.Sharding.Handlers.ContinuationTokens;
+using Raven.Server.Json;
+using Sparrow.Json;
+
+namespace Raven.Server.Documents.Operations
+{
+    internal class GetConflictsByEtagOperation : IMaintenanceOperation<GetConflictsResultByEtag>
```
Comment (karmeli87) [inline src/Raven.Server/Documents/Operations/GetReplicationConflictsOperation.cs]: `GetConflictsOperation`

File: src/Raven.Server/Documents/Operations/GetReplicationConflictsOperation.cs [hunk 13]
```diff
@@ -0,0 +1,107 @@
...
+    {
+        private readonly long _etag;
```
Comment (karmeli87) [inline src/Raven.Server/Documents/Operations/GetReplicationConflictsOperation.cs]: rename this to `start`

File: src/Raven.Server/Documents/Sharding/Commands/GetDocumentConflictsCommand.cs (hunk 4 above)
Comment (karmeli87) [reply src/Raven.Server/Documents/Sharding/Commands/GetDocumentConflictsCommand.cs]: ?

File: src/Raven.Server/Documents/Operations/GetReplicationConflictsOperation.cs (hunk 2 above)
Comment (karmeli87) [reply src/Raven.Server/Documents/Operations/GetReplicationConflictsOperation.cs]: rename `etag` to `start`

File: src/Raven.Server/Documents/Sharding/Handlers/Processors/Replication/ShardedReplicationHandlerProcessorForGetConflicts.cs [hunk 14]
```diff
@@ -0,0 +1,83 @@
...
+
+            public GetConflictsResultByEtag Combine(Memory<GetConflictsResultByEtag> results)
+            {
+                var span = results.Span;
+                var final = new GetConflictsResultByEtag();
+
+                final.Results = _handler.DatabaseContext.Streaming.PagedShardedItem(
+                    results,
+                    selector: r => r.Results,
+                    Comparer<ShardStreamItem<GetConflictsResultByEtag.ResultByEtag>>.Default,
```
Comment (karmeli87) [inline src/Raven.Server/Documents/Sharding/Handlers/Processors/Replication/ShardedReplicationHandlerProcessorForGetConflicts.cs]: let's sort it by last-modifed

File: src/Raven.Server/Documents/Sharding/Handlers/Processors/Replication/ShardedReplicationHandlerProcessorForGetConflicts.cs [hunk 15]
```diff
@@ -0,0 +1,83 @@
...
+            }
+
+            public HttpRequest HttpRequest => _handler.HttpContext.Request;
+
+            public GetConflictsResultByEtag Combine(Memory<GetConflictsResultByEtag> results)
+            {
+                var span = results.Span;
+                var final = new GetConflictsResultByEtag();
+
+                final.Results = _handler.DatabaseContext.Streaming.PagedShardedItem(
```
Comment (karmeli87) [inline src/Raven.Server/Documents/Sharding/Handlers/Processors/Replication/ShardedReplicationHandlerProcessorForGetConflicts.cs:66]: note to myself: move this method from the `Streaming` into a new partial class `Paging`

File: src/Raven.Server/Json/JsonDeserializationServer.cs [hunk 16]
```diff
@@ -227,6 +231,10 @@ internal sealed class JsonDeserializationServer : JsonDeserializationBase
 
         public static readonly Func<BlittableJsonReaderObject, LastChangeVectorForCollectionResult> LastChangeVectorForCollectionResult = GenerateJsonDeserializationRoutine<LastChangeVectorForCollectionResult>();
 
+        public static readonly Func<BlittableJsonReaderObject, BlittableArrayResult> BlittableArrayResult = GenerateJsonDeserializationRoutine<BlittableArrayResult>();
```
Comment (karmeli87) [inline src/Raven.Server/Json/JsonDeserializationServer.cs]: do you still need that?

File: test/SlowTests/Sharding/Client/Operations/GetDocumentConflictsTests.cs [hunk 17]
```diff
@@ -0,0 +1,212 @@
...
+{
+    public class GetDocumentConflictsTests : RavenTestBase
+    {
+        const string _suffix = "suffix";
+
+        public GetDocumentConflictsTests(ITestOutputHelper output) : base(output)
+        {
+        }
+
+        [RavenFact(RavenTestCategory.Voron)]
```
Comment (karmeli87) [inline test/SlowTests/Sharding/Client/Operations/GetDocumentConflictsTests.cs]: those are not `Voron` :)

**Comment by karmeli87** [comment]
I think this might require also some studio changes not to rely on the etag but instead use the `Start` & `PageSize` params

File: src/Raven.Server/Documents/Operations/GetReplicationConflictsOperation.cs (hunk 2 above)
Comment (shiranshalom) [reply src/Raven.Server/Documents/Operations/GetReplicationConflictsOperation.cs]: done

File: src/Raven.Server/Documents/Operations/GetReplicationConflictsOperation.cs (hunk 12 above)
Comment (shiranshalom) [reply src/Raven.Server/Documents/Operations/GetReplicationConflictsOperation.cs]: done

File: src/Raven.Client/Documents/Commands/GetConflictsResult.cs (hunk 10 above)
Comment (shiranshalom) [reply src/Raven.Client/Documents/Commands/GetConflictsResult.cs]: done

File: src/Raven.Client/Documents/Commands/GetConflictsResult.cs (hunk 11 above)
Comment (shiranshalom) [reply src/Raven.Client/Documents/Commands/GetConflictsResult.cs]: done

File: src/Raven.Server/Documents/Operations/GetReplicationConflictsOperation.cs (hunk 13 above)
Comment (shiranshalom) [reply src/Raven.Server/Documents/Operations/GetReplicationConflictsOperation.cs]: done

File: src/Raven.Server/Documents/Sharding/Commands/GetDocumentConflictsCommand.cs (hunk 4 above)
Comment (shiranshalom) [reply src/Raven.Server/Documents/Sharding/Commands/GetDocumentConflictsCommand.cs]: done

File: src/Raven.Server/Web/Studio/Sharding/Commands/GetConflictResolutionCommand.cs (hunk 7 above)
Comment (shiranshalom) [reply src/Raven.Server/Web/Studio/Sharding/Commands/GetConflictResolutionCommand.cs]: done

File: src/Raven.Server/Json/JsonDeserializationServer.cs (hunk 16 above)
Comment (shiranshalom) [reply src/Raven.Server/Json/JsonDeserializationServer.cs]: done

File: test/SlowTests/Sharding/Client/Operations/GetDocumentConflictsTests.cs (hunk 17 above)
Comment (shiranshalom) [reply test/SlowTests/Sharding/Client/Operations/GetDocumentConflictsTests.cs]: done

File: src/Raven.Server/Documents/Sharding/Handlers/Processors/Replication/ShardedReplicationHandlerProcessorForGetConflicts.cs (hunk 14 above)
Comment (shiranshalom) [reply src/Raven.Server/Documents/Sharding/Handlers/Processors/Replication/ShardedReplicationHandlerProcessorForGetConflicts.cs]: done

**Comment by Danielle9897** [comment]
@shiranshalom pls check 
When getting info for non-sharded/single shard then all results are returned (instead of pageSize)

File: src/Raven.Client/Documents/Commands/GetConflictsResult.cs [hunk 18]
```diff
@@ -20,4 +22,20 @@ public class Conflict
             public BlittableJsonReaderObject Doc { get; set; }
         }
     }
+
+    public class GetConflictsPreviewResult
```
Comment (ppekrol) [inline src/Raven.Client/Documents/Commands/GetConflictsResult.cs]: internal

File: src/Raven.Server/Documents/Handlers/Processors/Replication/AbstractReplicationHandlerProcessorForGetConflicts.cs [hunk 19]
```diff
@@ -0,0 +1,87 @@
...
+    internal abstract class AbstractReplicationHandlerProcessorForGetConflicts<TRequestHandler, TOperationContext> : AbstractHandlerProcessor<TRequestHandler, TOperationContext>
+        where TRequestHandler : RequestHandler
+        where TOperationContext : JsonOperationContext
+    {
+        protected AbstractReplicationHandlerProcessorForGetConflicts([NotNull] TRequestHandler requestHandler, [NotNull] JsonContextPoolBase<TOperationContext> contextPool)
+            : base(requestHandler, contextPool)
+        {
+        }
+
+        protected abstract Task<GetConflictsPreviewResult> GetConflictsPreviewAsync(TOperationContext context, long etag, int pageSize);
```
Comment (ppekrol) [inline src/Raven.Server/Documents/Handlers/Processors/Replication/AbstractReplicationHandlerProcessorForGetConflicts.cs]: this is start now, not etag?

File: src/Raven.Server/Documents/Operations/GetConflictsOperation.cs [hunk 20]
```diff
@@ -0,0 +1,106 @@
...
+                sb.Append($"{node.Url}/databases/{node.Database}/replication/conflicts");
+
+                if (_token != null)
+                    sb.Append($"?{ContinuationToken.ContinuationTokenQueryString}={Uri.EscapeDataString(_token)}");
+                else
+                {
+                    sb.Append($"?start={_start}");
+
+                    if (_pageSize.HasValue)
+                        sb.Append($"&pageSize={_pageSize}");
```
Comment (ppekrol) [inline src/Raven.Server/Documents/Operations/GetConflictsOperation.cs:87]: avoid sending this if this is int.MaxValue?

File: src/Raven.Server/Documents/Sharding/Handlers/ShardedDatabaseRequestHandler.ContinuationTokens.cs [hunk 21]
```diff
@@ -21,6 +21,14 @@ public ShardedPagingContinuation GetOrCreateContinuationToken(JsonOperationConte
                             new ShardedPagingContinuation(_handler.DatabaseContext, _handler.GetStart(), _handler.GetPageSize());
                 return token;
             }
+
+            public ShardedPagingContinuation GetOrCreateContinuationToken(JsonOperationContext context, int etag, int pageSize)
```
Comment (ppekrol) [inline src/Raven.Server/Documents/Sharding/Handlers/ShardedDatabaseRequestHandler.ContinuationTokens.cs]: etag?

File: src/Raven.Server/Web/Operations/GetSuggestConflictResolutionOperation.cs [hunk 22]
```diff
@@ -0,0 +1,57 @@
...
+            _documentId = documentId;
+        }
+
+
+        public RavenCommand<ConflictResolverAdvisor.MergeResult> GetCommand(DocumentConventions conventions, JsonOperationContext context)
+        {
+            return new GetSuggestConflictResolutionCommand(_documentId);
+        }
+
+        public class GetSuggestConflictResolutionCommand : RavenCommand<ConflictResolverAdvisor.MergeResult>
```
Comment (ppekrol) [inline src/Raven.Server/Web/Operations/GetSuggestConflictResolutionOperation.cs:27]: can be made private?

File: src/Raven.Server/Documents/Handlers/Processors/Replication/AbstractReplicationHandlerProcessorForGetConflicts.cs (hunk 19 above)
Comment (ppekrol) [reply src/Raven.Server/Documents/Handlers/Processors/Replication/AbstractReplicationHandlerProcessorForGetConflicts.cs]: ValueTask?

**Comment by ppekrol** [comment]
Can you rebase?

**Comment by shiranshalom** [comment]
> @shiranshalom pls check When getting info for non-sharded/single shard then all results are returned (instead of pageSize)

Done

File: src/Raven.Client/Documents/Commands/GetConflictsResult.cs (hunk 18 above)
Comment (shiranshalom) [reply src/Raven.Client/Documents/Commands/GetConflictsResult.cs]: done

File: src/Raven.Client/Documents/Commands/GetConflictsCommand.cs (hunk 9 above)
Comment (shiranshalom) [reply src/Raven.Client/Documents/Commands/GetConflictsCommand.cs]: This file is outdated - the command moved to Server-side

File: src/Raven.Server/Documents/Handlers/Processors/Replication/AbstractReplicationHandlerProcessorForGetConflicts.cs (hunk 19 above)
Comment (shiranshalom) [reply src/Raven.Server/Documents/Handlers/Processors/Replication/AbstractReplicationHandlerProcessorForGetConflicts.cs]: done

File: src/Raven.Server/Documents/Operations/GetConflictsOperation.cs (hunk 20 above)
Comment (shiranshalom) [reply src/Raven.Server/Documents/Operations/GetConflictsOperation.cs:87]: done

File: src/Raven.Server/Documents/Sharding/Handlers/ShardedDatabaseRequestHandler.ContinuationTokens.cs (hunk 21 above)
Comment (shiranshalom) [reply src/Raven.Server/Documents/Sharding/Handlers/ShardedDatabaseRequestHandler.ContinuationTokens.cs]: done

File: src/Raven.Server/Web/Operations/GetSuggestConflictResolutionOperation.cs (hunk 22 above)
Comment (shiranshalom) [reply src/Raven.Server/Web/Operations/GetSuggestConflictResolutionOperation.cs:27]: @ppekrol No, I need it for `ShardedGetReplicationConflictsOperation` - but I can move the entire class (`GetSuggestConflictResolutionOperation`) to `ShardedReplicationHandlerProcessorForGetConflicts.cs`. Thoughts? :-)

Comment (ppekrol) [reply src/Raven.Server/Web/Operations/GetSuggestConflictResolutionOperation.cs:27]: no need

Comment (shiranshalom) [reply src/Raven.Server/Web/Operations/GetSuggestConflictResolutionOperation.cs:27]: OK, so done :)

File: src/Raven.Server/Documents/Operations/GetConflictsOperation.cs (hunk 20 above)
Comment (ppekrol) [reply src/Raven.Server/Documents/Operations/GetConflictsOperation.cs:87]: default(int) = 0, not int.MaxValue

Comment (shiranshalom) [reply src/Raven.Server/Documents/Operations/GetConflictsOperation.cs:87]: done

**Pull request merged**